8.1/startup/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Express-driven static site hosted from default App Service wwwroot directory",
   "main": "default-static-site.js",
   "dependencies": {
+    "applicationinsights": "0.21.0",
```

Comment:
Additionally, while I used a static version for this dependency, I'm cool with making it pick up patch changes if we thought that made sense. Since these images would need to be rebuilt to pull in any updated dependencies anyways, I figured it made sense to just be explicit about the version being bundled. Would love feedback though.